src/checking/issue_mapping/issue_mapping.test.tsx

```diff
@@ -0,0 +1,89 @@
+// IMPORTS
+import * as ISSUE from "./issue_mapping";
+import { IssueObject } from "./issue_mapping";
+
+test("creating a issueObject", () => {
+	const severityNumber = ISSUE.Severity.Warning;
+	const issueJSONObjectKey = ISSUE.knownIssueCodes.WARNINGS.MISSING_STATEMENT_AT_THE_END;
+	const issueCode = ISSUE.getIssueCodeFromJSON(issueJSONObjectKey);
+	const issueMessage = issueJSONObjectKey.MISSING_STATEMENT_AT_THE_END;
+	const positionNumber = 200;
+	const issue: IssueObject = ISSUE.createIssue(issueCode, issueMessage, severityNumber, positionNumber);
+
+	const resultCode = issue.code;
+	const expectedResultCode = "MISSING_STATEMENT_AT_THE_END";
+	const resultMessage = issue.message;
+	const expectedResultMessage = "It seems, you forgot some statements at the end of your expression.";
+	const resultSeverity = issue.severity;
+	const expectedResultSeverity = 1;
+	const resultPosition = issue.position;
+	const expectedResultPosition = 200;
+
+	expect(resultCode).toEqual(expectedResultCode);
+	expect(resultMessage).toEqual(expectedResultMessage);
+	expect(resultSeverity).toEqual(expectedResultSeverity);
+	expect(resultPosition).toEqual(expectedResultPosition);
+
+});
+
+test("creating a issueObject that is that is unknown", () => {
+	const severityNumber = ISSUE.Severity.FatalError;
+	const issueCode = "A-TEST";
+	const issueMessage = "just a test message";
+	const positionNumber = NaN;
+	const issue: IssueObject = ISSUE.createIssue(issueCode, issueMessage, severityNumber, positionNumber);
+
+	const resultCode = issue.code;
+	const expectedResultCode = "A-TEST";
+	const resultMessage = issue.message;
+	const expectedResultMessage = "just a test message";
+	const resultSeverity = issue.severity;
+	const expectedResultSeverity = 3;
```

Comment:
Since we didn't specify the enum-values, you shouldn't test on them here. These tests will easily break if someone reorders the severities or adds new ones. Compare with `Severity.FUBAR` instead.